modules/edu.gemini.seqexec.engine/src/main/scala/edu/gemini/seqexec/engine/Execution.scala

```diff
@@ -77,6 +77,10 @@ object Execution {
 sealed trait Result
 
 object Result {
-  case class OK[R](r: R) extends Result
+  case class OK(response: Response) extends Result
   case class Error[E](e: E) extends Result
+
+  sealed trait Response
+  case class Configured(r: String) extends Response
+  case class Observed(fileId: FileId) extends Response
```

Comment:
I think we are missing one here, there should be a `StartObserve` and contain the `fileId` and an estimate of the running time (That can be blank for now)